src/main/java/com/gmail/goosius/siegewar/utils/PermissionUtil.java

```diff
@@ -54,7 +54,6 @@ public static boolean hasNationMilitaryRank(Resident resident) {
 		return resident.isKing() || nationMilitaryRanks.stream().anyMatch(rank -> resident.getNationRanks().contains(rank));
 	}
 
-	//LP Glitch mitigation (TODO - remove this pattern by resolving on the lp config side)

```

Comment:
:question: 
- Hi should we keep this line?
- This method is still, after all, a workaround from the normal mechanics, specifically do deal with a glitch in LuckyPerms.
- In terms of updating the comment, Id rather find out what the glitch is and add that to the comment  (or if the glitch is actually gone now, to remove it altogether in a different PR) .

Reply:
The glitch seems to be gone after I did some testing, so I've removed it altogether.